components/bl-stepper-component/src/index.js

```diff
@@ -0,0 +1,70 @@
+import { useMemo, useState, useEffect } from 'react';
+import { StepperItem } from './components/stepper-item';
+import { useStepperClassName } from './use-stepper-class-name'
+
+export default function MyCustomComponent({ component, eventHandlers }) {
+  const { stepList, display, stepperType, countSteps } = component;
+  const { onStepChange } = eventHandlers;
+  const [currentStep, setCurrentStep] = useState(0);
+  const [completedSteps, setCompletedSteps] = useState([]);
+  const stepperClassName = useStepperClassName(stepperType);
+
+  useEffect(() => {
+    onStepChange();
+  }, [currentStep]);
+
+  const steps = useMemo(() => {
+    console.log(stepList);
+    if (!stepList) {
+      console.log('createArray', countSteps);
+      return new Array(countSteps).fill(' ');
+    }
+
+    if (typeof stepList === 'string') {
+      return stepList.split(',');
+    }
+
+    return stepList;
+  }, [stepList]);
+
+  component.goNextStep = () => {
+    if (currentStep !== steps.length) {
+      setCurrentStep((prev) => prev + 1);
```

Comment:
Isn't be better if we store list of objects? Rather than two lists.

` [{content: 'step text', completed: true}]`